config/locales/en/admin.yml

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
If the `admin` keys are being separated out, then you should add the key to `config/i18n-tasks.yml` so that if we ever run `i18n-tasks normalize -p`, it will keep the `admin` keys in a separate file.

Reply:
I've modified `config/i18n-tasks.yml` following the [README](https://github.com/glebm/i18n-tasks#pattern-router), so that we no longer need to manually add new filenames to the config file.

Reply:
Nice!  But it does mean  that one file is created for each level-1 key, so 11 new files are created.  Are we happy to leave it like that and run `18n-tasks normalize -p` sometime in the future, or run it now and check the new files into github?

I'm easy either way.

Reply:
I ran `i18n-tasks normalize` to check my modification worked. - this PR includes the modifications made by running that command after I made the change to the config..

Reply:
Just had another play `i18n-tasks normalize -p` does add new files. Personally, I'd prefer a system that does require me to keep updating the configuration, even if that does generate more files.